Calc2.py

```diff
@@ -0,0 +1,22 @@
+with open('Calcs.txt', 'r') as calcs_file:
+    contents = calcs_file.read()
+
+lines=contents.splitlines()
+
+count=0
+for x in lines:
+    line=lines[count].split(",")
+    line2=line[0].split(" ")
+    Operation = (line2[1])
+    Number1 = int((line2[2]))
+    Number2 = int((line2[3]))
+    count +=1
```

Comment:
I think we could simplify this a little:
* Your loop is already picking out one line at a time (as `x`) which we then ignore!
* You split by `,` even though there aren't any commas
* The variable names could probably be a bit clearer

So maybe something like:
```suggestion
for line in lines:
    line_parts=line.split(" ")
    Operation = (line_parts[1])
    Number1 = int((line_parts[2]))
    Number2 = int((line_parts[3]))
```